schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -54,6 +54,18 @@
           },
           "minItems": 1,
           "maxItems": 10
+        },
+        "architecture": {
+          "description": "The architecture for a package install",
+          "type": "string",
```

Comment:
Shall we consider making it a list to be more flexible(like local above) ?

Reply:
The ticket made it sound like you’d select a preferred (single) architecture instead of multiple, but I can do that.

Reply:
I was also thinking we would have a "single" preferred or required architecture. I can't think of a valid scenario where there would be three architectures that could all work and you would prefer (a), accept (b), and reject (c). With locale, I could see a set you would like in some kind of order so you could essentially have a bit of control when one of the languages you used was available, but it wasn't a logical close match otherwise. Let's keep this as a singleton for now.

Reply:
Just after I hit send, I remembered some emulation might be valid, but I think the "requires" forces it to be what I want and "prefers" will take anything else.

Reply:
That is how it works. Prefers will fallback to the normal logic if the architecture you prefer is not available. If your required architecture is not available you get an error.

Reply:
Yeah, I was just not completing thoughts last night. Days have bee feeling like weeks lately. I'm trying to close a lot of gaps. We have some heavy investments coming in related to Intune integration for the replacement of store for business, and Win32 support for the store. The next couple of quarters are going to be busy.

Reply:
I would rather have it be an array to start.  At some point someone will want an array and then we will have two settings (or have to implement it allowing both scalar and array).  Even requirements makes sense to support multiple, just like locales.  I might be an ARM purist on my ARM64 machine and so require `{ arm64, arm }`.